### ISSUE TITLE
Better Fluid Support!!!

### ISSUE THREAD

**Issue by 7OOTnegaTerces** [raoulvdberge/refinedstorage#932]
#### Issue description:
First, I'm starting to use a lot of fluids and it's getting really old having to run all the way back to my stationary fluid monitor EVERY SINGLE TIME I want to know how much fluid I have in my system.  It's MOST DEFINITLY getting in the way of debugging my setups when something's going wrong (or even knowing that something's going wrong in the first place)!  Could you PLEASE implement a wireless fluid monitor?!?  Honestly, I've no idea why you don't have one already!

Second, your fluid monitor is great and all that, but besides being stationary, it's also completely inert.  Want a bucket of water, milk, or anything else?  Well whip out a constructor and get it yourself!  Same goes for emptying a bucket as well!  The fluid monitor is, in my opinion, the least useful block in this mod because it does TOO LITTLE.  You can input and remove items from the regular item grid, would you PLEASE enable the fluid grid to do the same!!!

Third, if you're going to implement points one and two, then why not let the wireless fluid monitor empty and fill buckets as well.  And if you're going to do that, you might as well create a wireless bucket that can tap directly into the system (in addition to letting the wireless fluid monitor fill buckets, of course)!!!

Fourth, don't forget about bucketless fluid support!!!  In fact point three should solve this automatically, since if you implemented a wireless bucket and made it so that you could fill it, i.e. set it's fluid type, from the (wireless) fluid monitor, that would include otherwise bucketless fluids, giving you a bucket to use when setting fluid selection fields and solve the whole issue!!!  Or you could just use regular buckets filled from the (wireless) fluid monitor, if it's possible for you to provide support for buckets with fluid from other mods that don't provide buckets themselves.

If you feel that wireless buckets are too OP (though I'd have to ask WHY, since it just giving you the same access to your fluids that you already have with your items), you could still solve the bucketless fluid issue by creating a special 'dummy' RS bucket that can be set from the wireless fluid monitor (if you don't include wireless support...I'm going to get really grumpy and tired running back and forth constantly) that's just for setting fluid selection fields and can't actually place any fluids in the world itself.

- Minecraft: 1.10.2
- Forge:2185
- Refined Storage: 1.2.12

**Comment by 7OOTnegaTerces** [comment]
After my lastest update to first post of this issue, I'm really wondering if I should I close my other thread on Bucketless Fluid Support...

**Comment by raoulvdberge** [comment]
You sound really rude.

**Comment by 7OOTnegaTerces** [comment]
Sorry, that was not my intention. Text is a poor medium for conveying emotion...  I will admit that the impetus for writing this post in the first place was frustration, which may have come across...poorly.  Do you want me to rewrite my first post (in the morning)?

**Comment by raoulvdberge** [comment]
Please do.

**Comment by PONYRYDE** [comment]
@7OOTnegaTerces While this is definitely more entertaining than a bug report I am guessing your Fluid Grid doesn't take too kindly to be called a USELESS STATIONARY MONITOR as mine gives me buckets of fluid for the asking. Perhaps you should try a different approach.

Here by accident but couldn't resist.

**Comment by raoulvdberge** [comment]
> Could you PLEASE implement a wireless fluid monitor?!? Honestly, I've no idea why you don't have one already!

The 1.11 version will have a wireless fluid grid, in fact, it's already implemented and completed.

> Second, your fluid monitor is great and all that, but besides being stationary, it's also completely inert. Want a bucket of water, milk, or anything else? Well whip out a constructor and get it yourself! 

False.

https://refinedstorage.raoulvdberge.com/wiki/fluid-grid

"To insert a fluid, players can SHIFT click or just click in a filled fluid container."
"Players can extract the fluid by clicking on it. The Fluid Grid will then fill a bucket with the selected fluid."
"Refined Storage will first search an empty bucket from your storage system. If none is found, a bucket will be searched in your inventory."

> Third, if you're going to implement points one and two, then why not let the wireless fluid monitor empty and fill buckets as well. 

That already works in 1.11, as I said.

> And if you're going to do that, you might as well create a wireless bucket that can tap directly into the system (in addition to letting the wireless fluid monitor fill buckets, of course)!!!

No!!!

> Fourth, don't forget about bucketless fluid support!!!

I won't!!!

> If you feel that wireless buckets are too OP (though I'd have to ask WHY, since it just giving you the same access to your fluids that you already have with your items), you could still solve the bucketless fluid issue by creating a special 'dummy' RS bucket that can be set from the wireless fluid monitor

Sorry, but this idea is dumb.

In conclusion: 85% / 90% of your issue report is fake news. Alternative facts. Literally Trump.
